sale_medical_prescription/models/medical_medicament.py

```diff
@@ -1,30 +1,29 @@
 # -*- coding: utf-8 -*-
-# © 2016 LasLabs Inc.
+# Copyright 2016 LasLabs Inc.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import fields, models, api
+from openerp import api, fields, models
 
 
 class MedicalMedicament(models.Model):
     _inherit = 'medical.medicament'
+
     is_prescription = fields.Boolean(
         string='Prescription Required',
         compute='_compute_is_prescription',
-        help='Check this if a prescription is required for this medicament',
+        help='Indicates if a prescription is required for this medicament',
     )
     is_controlled = fields.Boolean(
         string='Controlled Substance',
         help='Check this if the medicament is a controlled substance',
     )
 
     @api.multi
-    def _compute_is_prescription(self, ):
+    def _compute_is_prescription(self):
         prescription_categ_id = self.env.ref(
-            'medical_prescription_sale.product_category_rx'
+            'sale_medical_prescription.product_category_rx'
         )
         for rec_id in self:
-            if not rec_id.categ_id:
```

Comment:
`categ_id` is a required field AFAIK so removed this check. @lasley pointing out in case you want me to keep it.